From d3be47eddfba095e1797750416a32a2d7aa799f9 Mon Sep 17 00:00:00 2001
From: begasus <begasus@gmail.com>
Date: Tue, 16 May 2023 19:28:02 +0200
Subject: Fixes: gettext infrastructure mismatch: using a Makefile.in.in from
 gettext version 0.19 but the autoconf macros are from gettext version 0.20


diff --git a/configure.ac b/configure.ac
index 2043c3d..e11efbb 100644
--- a/configure.ac
+++ b/configure.ac
@@ -40,6 +40,7 @@ AC_SUBST([ALL_LINGUAS], "_BF_LINGUAS")
 GETTEXT_PACKAGE=AC_PACKAGE_NAME
 AC_SUBST(GETTEXT_PACKAGE)
 AC_DEFINE_UNQUOTED([GETTEXT_PACKAGE], ["$GETTEXT_PACKAGE"], [gettext domain])
+AM_GNU_GETTEXT_REQUIRE_VERSION([0.19])
 AM_GLIB_GNU_GETTEXT
 
 IT_PROG_INTLTOOL
-- 
2.37.3

